imports/plugins/core/ui-navbar/client/components/navbar.js

```diff
@@ -4,6 +4,7 @@ import { Components } from "@reactioncommerce/reaction-components";
 import { Meteor } from "meteor/meteor";
 import { Reaction } from "/client/api";
 import startTour from "/imports/plugins/custom/vendor-tour/client/vendorTour";
+import StartCustomerTour from "/imports/plugins/custom/customer-tour/client/customerTour";
```

Comment:
@Rhotimee, good work, however I see that  StartCustomerTour is in pascal case, please let's be consistent on our naming conventions.